src/strings/en-us.json

```diff
@@ -1492,5 +1492,8 @@
     "EveryXMinutes": "Every {0} minutes",
     "EveryHour": "Every hour",
     "EveryXHours": "Every {0} hours",
-    "OnApplicationStartup": "On application startup"
+    "OnApplicationStartup": "On application startup",
+    "ExperimentalBlurayPlayback": "Blu-Ray playback is experimental. Jellyfin cannot decrypt content protected by DRM. All the titles in the Blu-Ray will be played anyway, including the protected ones. Some parts might be in black",
```

Comment:
```suggestion
    "ExperimentalBlurayPlayback": "Jellyfin cannot decrypt content protected by DRM but all content will be attempted regardless, including protected titles. This may cause some files to appear completely black during the encrypted sections.",
```

Reply:
I would change all three to match this warning instead.

Reply:
@dkanada But what if one title is interactive and not encrypted? They can't be played back, but they're not encrypted. Black Playback ≠ Content Encrypted

 Also, I would use **titles** instead of **files**

Reply:
Interactive DVD titles will simply show up as short videos. Blu-ray menus aren't really titles per se, so nothing interactive will show up (afaik all interactive content on Blu-ray discs uses Java)

Reply:
@MrTimscampi You tested that? Because I know VLC and so on plays them as short videos, but from what I guessed from the original message (not @Artiume's) is that those titles doesn't work at all in Jellyfin.